lib/confirmation_text_sender.rb

```diff
@@ -0,0 +1,24 @@
+require 'rubygems'
+require 'twilio-ruby'
+require 'time'
+
+class ConfirmationTextSender
+
+  def send_message(message)
+
+    account_sid = 'AC9418dc2fc0bb5cbb33e505c227d3a3c1'
+    auth_token = '5490266efdf7640c6e1d59a7da563967'
```

Comment:
Don't commit auth_tokens and secrets to public repos.  You ought to regenerate this in Twilio to be on the safe side.